install.sh

```diff
@@ -64,7 +80,17 @@ fi
 
 $FETCH $SOURCE_URL > $BOOTSTRAP_DIR/fpm.F90
 
-$FC $FFLAGS -J $BOOTSTRAP_DIR $BOOTSTRAP_DIR/fpm.F90 -o $BOOTSTRAP_DIR/fpm
+if command -v pushd > /dev/null 2>&1; then
+    PUSHD=pushd
+    POPD=popd
+else
+    PUSHD=my_pushd
+    POPD=my_popd
+fi
+
+$PUSHD $BOOTSTRAP_DIR
+    $FC $FFLAGS fpm.F90 -o fpm
+$POPD
```

Comment:
What's wrong with using `cd`?

```suggestion
ROOT_DIR=$PWD
cd $BOOTSTRAP_DIR
$FC $FFLAGS fpm.F90 -o fpm
cd "$ROOT_DIR"
```

Reply:
You shouldn't `cd ..` in a script, you don't know where you will end up. The additional line for saving the directory seems clearer to me than a custom function which we do not test.

Reply:
Committed, thanks!